ArcGISRuntimeSDKQt_QMLSamples/Maps/MinMaxScale/README.md

```diff
@@ -0,0 +1,19 @@
+# Min Max Scale
+
+This demonstrates how to set the minimum and maximum scale of an ArcGIS Map.
```

Comment:
update to "This sample demonstrates how to set the minimum and maximum scale of a map."